include/benchmark/benchmark.h

```diff
@@ -167,21 +167,23 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 #define BENCHMARK_HAS_CXX11
 #endif
 
-#include <stdint.h>
-
-#include <algorithm>
-#include <cassert>
-#include <cstddef>
-#include <iosfwd>
-#include <map>
-#include <set>
-#include <string>
-#include <vector>
+#include <stdint.h>  // IWYU pragma: keep
+
+#include <algorithm>  // for max
+#include <cassert>    // for assert
+#include <cstddef>    // for size_t, ptrdiff_t
+#include <iosfwd>     // for ostream
+#include <iterator>   // for forward_iterator_tag
+#include <map>        // for operator!=, map<>::mapped_type, map<>::const_...
+#include <set>        // for set
+#include <string>     // for string
+#include <vector>     // for vector
+// IWYU pragma: no_include <cstdint>
```

Comment:
What's this here for?

Reply:
`// IWYU pragma: no_include .*` tell IWYU that it is wrong and it should not suggest to include said `.*` here.
`// IWYU pragma: keep` is the other way around, that it should not suggest to delete it.
I could drop these annotations, but they will be needed the very next time anyone will
want to run IWYU again, it would be much harder to keep things correct without them.